Automattic/wp-calypso
# Automattic for Agencies: Update initial loading logo

Resolves https://github.com/Automattic/automattic-for-agencies-dev/issues/203

## Proposed Changes

This PR updates the loading logo for A4A to use the Automattic logo instead of the WordPress logo.

## Testing Instructions

- Open the Calyso live link for A4A.
- Verify that the initial loading logo looks like below:

<img width="441" alt="Screenshot 2024-04-08 at 6 34 07 PM" src="https://github.com/Automattic/wp-calypso/assets/10586875/ad183c2e-e850-4afa-a047-9ff7df06b025">


## Pre-merge Checklist
